docs/datadoc/fetching-resources-from-a-triplestore.md

```diff
@@ -0,0 +1,118 @@
+Working with already documented resources
+=========================================
+
+The [tripper.datadoc] module also includes functionality for easy searching of the documented reseources.
+
+For these examples there must be a triplestore instance available, poplated with some data.
+```python
+>>> from tripper import Triplestore
+>>> from tripper.datadoc import save_datadoc
+>>> ts = Triplestore(backend="rdflib")
+>>> save_datadoc(ts,"https://raw.githubusercontent.com/EMMC-ASBL/tripper/refs/heads/master/tests/input/semdata.yaml") # doctest: +ELLIPSIS
+{'@graph': [...], ...}
+
+```
+
+Searching the knowledge base
+----------------------------
+
+### Get all IRIs of all datasets in the kb
+
+```python
+>>> from tripper.datadoc import search
+>>> search(ts) # doctest: +ELLIPSIS
+[...]
+
+```
+
+This will return a list of all datasets in the knowledge base.
+
+
+### Search with filtering criteria
+
+Before adding specific filtering criteria it is importa to bind prefixes to the triplestore instance:
+
+```python
+>>> DCAT = ts.bind("dcat", "http://www.w3.org/ns/dcat#")
+>>> DCTERMS = ts.bind("dcterms", "http://purl.org/dc/terms/")  
```

Comment:
The intention is that you should not need to bind standard prefixes defined in the keywords file. Have you tested to rerun the examples without these bindings? If it works as intended, we should change this sentence to something like

```suggestion
Before adding specific filtering criteria it is important to bind non-standard prefixes to corresponding namespaces (standard prefixes defined in the keywords file, like dcterms, dcat, etc does not need to be defined again):

```python
>>> DATA = ts.bind("data", "http://example.com/data#")
>>> MAT = ts.bind("mat", "http://example.com/materials#")

```